package.json

```diff
@@ -22,7 +22,6 @@
     "readme": "./bin/plugin/cli.js readme",
     "translations": "./bin/plugin/cli.js translations",
     "enabled-modules": "./bin/plugin/cli.js enabled-modules",
-    "benchmark": "./bin/plugin/cli.js benchmark",
```

Comment:
We also need to remove `autocannon`, `table` and `csv-stringify` dependencies.

Reply:
@eugene-manuilov Good catch, updated!